### ISSUE TITLE
"All tests passed!" reported when not true

### ISSUE THREAD

**Issue by eraserhd** [koute/cargo-web#113]
From [parinfer-rust](https://github.com/eraserhd/parinfer-rust).

If I add an `assert_eq!("x","y");` to the beginning of the `tests/cases.rs:paren_mode()` test, I can do this:
```
[jfelice@C02PJ2PAFVH8 parinfer-rust(master)]$ cargo +nightly web test --nodejs
warning: debug builds on the wasm32-unknown-unknown are currently totally broken
         forcing a release build
    Finished release [optimized] target(s) in 0.06s
warning: debug builds on the wasm32-unknown-unknown are currently totally broken
         forcing a release build
    Finished release [optimized] target(s) in 0.06s
All tests passed!
[jfelice@C02PJ2PAFVH8 parinfer-rust(master)]$ node target/wasm32-unknown-unknown/release/cases-96b0933d50a4f078.js
wasm://wasm/0038a65e:1269

^

RuntimeError: unreachable
    at __rust_start_panic (wasm-function[1268]:33)
    at rust_panic (wasm-function[1260]:30)
    at std::panicking::rust_panic_with_hook::hda7da2195919e0a1 (wasm-function[1257]:436)
    at std::panicking::continue_panic_fmt::h7f061c9d3b21c89e (wasm-function[1253]:121)
    at std::panicking::begin_panic_fmt::h864132dbee504c33 (wasm-function[1256]:93)
    at cases::paren_mode::h2855760fdcdf90b7 (wasm-function[61]:133)
    at core::ops::function::FnOnce::call_once::hbc83092abee3a7b6 (wasm-function[93]:13)
    at test::__rust_begin_short_backtrace::h95180c9599a2d2dd (wasm-function[152]:3)
    at <F as alloc::boxed::FnBox<A>>::call_box::h681ac536053b168b (wasm-function[151]:6)
    at std::panicking::try::do_call::h1b5228b9357c95d3 (wasm-function[364]:14)
```

Looks like I have node `v9.5.0`, and I just ran `rustup update` and upgraded `cargo-web`.

**Comment by koute** [comment]
This happens because you have multiple crate types in your `Cargo.toml` so `cargo` reports multiple generated artifacts after building, and we run the first one.

Fixed on `master`; thank for the report!

**Comment by eraserhd** [comment]
@koute Thank you!  🎉

**Comment by koute** [comment]
Released `0.6.14` with this fix.
